public/app/features/dashboard-scene/saving/SaveDashboardForm.tsx

```diff
@@ -128,6 +128,15 @@ export function SaveDashboardForm({ dashboard, drawer, changeInfo }: Props) {
   return (
     <Stack gap={2} direction="column">
       <SaveDashboardFormCommonOptions drawer={drawer} changeInfo={changeInfo} />
+      {hasMigratedToV2 && (
+        <Alert title="Dashboard drastically changed" severity="warning">
```

Comment:
drastic!!!

Reply:
I could say ‘significantly,’ but my Spaniard accent tends to be more dramatic 😆